frontend/packages/dev-console/src/components/pipelines/detail-page-tabs/pipeline-details/PipelineVisualizationTask.tsx

```diff
@@ -104,8 +104,8 @@ const TaskComponent: React.FC<TaskProps> = ({
   const stepStatusList: StepStatus[] = stepList.map((step) => createStepStatus(step, status));
   const showStatusState: boolean = isPipelineRun && !!status && !!status.reason;
   const visualName = name || _.get(task, ['metadata', 'name'], '');
-  const path = pipelineRun
-    ? `${resourcePathFromModel(PipelineRunModel, pipelineRun, namespace)}/logs/${name}`
+  const path = pipelineRunName
+    ? `${resourcePathFromModel(PipelineRunModel, pipelineRunName, namespace)}/logs/${name}`
     : undefined;
```

Comment:
do we need to keep the undefined?

Reply:
It appears not, but this was pre-existing code. My change was just to change the variable away from `pipelineRun` because that implicitly means the K8s resource and not just the name.